packages/protocol/contracts/bridge/Bridge.sol

```diff
@@ -46,7 +46,10 @@ contract Bridge is EssentialContract, IBridge {
         address preferredExecutor;
     }
 
-    uint256 internal constant PLACEHOLDER = type(uint256).max;
+    // The slot in transient storage of the call context
+    // This is the keccak256 hash of "bridge.ctx_slot"
+    bytes32 private constant _CTX_SLOT =
+        0xe4ece82196de19aabe639620d7f716c433d1348f96ce727c9989a982dbadc2b9;
```

Comment:
Now uses transient storage to store the call context. Depending on the transaction, this may or may not save gas because transient storage does not have the refund issue like with normal storage. So the good thing about it is that it always reduces the max amount of gas needed in a transaction, and it doesn't depend on how much can be refunded.